### PR TITLE
chore: Add metadata column to Postgres media table

### PR DESCRIPTION
**Pull request by tomthornton**

### CODE REVIEW

File: packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  });
+}
+
+async function down({ context: queryInterface }) {
+  await queryInterface.removeColumn('media', 'metadata');
+}
+
+const name = '004-add_metadata_column';
+
+module.exports = { up, down, name, order: 6 };
```
Comment (vinnyjth) [inline packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js:17]: `AddIdToMedia003` has order `6` :) 
```suggestion
module.exports = { up, down, name, order: 7 };
```

File: packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+
+async function up({ context: queryInterface }) {
+  await queryInterface.addColumn('media', 'metadata', {
+    type: DataTypes.JSONB,
```
Comment (vinnyjth) [inline packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js:5]: Awesome idea having it `allowNull: false`   but also giving it a default value!

File: packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js (hunk 1 above)
Comment (tomthornton) [reply packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js:17]: @vinnyjth ahh got it. Thanks!

**Review by vinnyjth — APPROVED** [review]
Looks good, thanks Tom!

**Pull request merged**